trunner/tools/phoenix.py

```diff
@@ -101,9 +115,9 @@ def _reader(self):
 
     def _run(self):
         try:
-            wait_for_dev(self.port, timeout=10)
-        except TimeoutError as exc:
-            raise PhoenixdError(f"couldn't find {self.port}") from exc
+            self.port = wait_for_vid_pid(self.vid, self.pid, timeout=1)
```

Comment:
You've changed timeout from 10s to 1s, is this intended? Maybe make it a separate commit so git log would clearly show that not only the lookup mechanism has changed but also the timeout itself.

Reply:
Sorry, that was a mistake, I used this timeout to speed up debugging. We probably want to wait this 10 sec. because there is an option that the USB port will raise slower after some changes.